plugins/plugin_utils/action_module.py

```diff
@@ -69,9 +69,9 @@
     # For ansible-core 2.11, we can use the ArgumentSpecValidator. We also import
     # ModuleArgumentSpecValidator since that indicates that the 'classical' approach
     # will no longer work.
-    from ansible.module_utils.common.arg_spec import (
+    from ansible.module_utils.common.arg_spec import (  # noqa: F401, pylint: disable=unused-import
```

Comment:
Whatever you decide in [community.crypto#569](https://github.com/ansible-collections/community.crypto/pull/569#discussion_r1098527315), apply the same here.

Reply:
I've added a comment in e65b55327d7b34b7b93e7b32731b0560989e9165.

Reply:
/resolved